frontend/dist/index.mak

```diff
@@ -111,14 +110,19 @@
     <script type="text/javascript" src="/assets/bootstrap/popper-1.12.3.js?token=${cache_token}" nonce="${csp_nonce}"></script>
     <script type="text/javascript" src="/assets/bootstrap/bootstrap-4.0.0-beta.2.js?token=${cache_token}" nonce="${csp_nonce}"></script>
 
+    <script type="text/javascript" src="/assets/tracim.app.js?token=${cache_token}" nonce="${csp_nonce}"></script>
+
```

Comment:
What is the reason behind the move of this <script> tag? Adding a note explaining that this script must be loaded here could avoid unwanted changes later on (if the order is important for example).

Reply:
It was to fix an inconsistency during my tests, but I shouldn't have kept. Will redo some tests without the change and remove it if it's ok

Reply:
After some tests I'm a bit confused. The issue I ran into is that, with the previous position of this line (in the middle of the scripts tags), sometimes, apps aren't loaded when frontend tries to call them. Using pushpin server.

I tried back on develop and I'm having the same issue. I can reproduce it consistently (not 100% though).

So I tried the same on unstable.tracim.fr but I'm unable to reproduce it.

Reply:
So I'm reverting this fix
But we might need to keep an eye on it

Reply:
fix